pkg/gitreceive/k8s_util.go

```diff
@@ -241,7 +241,6 @@ func progress(msg string, interval time.Duration) chan bool {
 				return
 			case <-tick:
 				fmt.Println(msg)
-			default:
```

Comment:
there is select inside an empty for loop and if there is a default case doing nothing which means the for loop gets run continuously for no reason